worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactory.java

```diff
@@ -389,4 +393,158 @@ private void forwardAvailableJobs(final List<JobTrackingWorkerDependency> jobDep
             LOG.error("Error retrieving Dependent Job Info from the Job Service Database {}", e);
         }
     }
+
+    @Override
+    public void processTasks(final BulkWorkerRuntime bwr) throws InterruptedException
+    {
+        // Configured to wait 10 seconds between two batches
+        final long maxWaitingTime = Long.getLong(System.getenv("MAX_BATCH_WAITING_TIME"));
```

Comment:
You need to look at the JavaDoc view when programming.

![image](https://user-images.githubusercontent.com/20627297/95533627-89d72c00-09db-11eb-9a1a-ed3489e486c5.png)

This function takes the name of a system property as an argument, so it clearly doesn't do what you're intending.

Reply:
done